src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -16,6 +16,7 @@
 
 package com.google.codeu.data;
 
+import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
Sorry for the drive-by comment. Fwiw Google's Java style guide recommends putting import statements in alphabetical order: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

I don't feel super strongly about this, so it's up to you whether you want to fix this now or just include this in your next change... or to ignore it entirely.

Reply:
Just fixed it and added it to this PR! 

I was wonderig how they were ordered. In the past I've seen grouped by prefix and then by length or use, but alphabetical makes sense.